src/patternfly/components/DescriptionList/description-list.scss

```diff
@@ -23,6 +23,20 @@ $pf-c-description-list--breakpoint-map: build-breakpoint-map("base", "sm", "md",
   --pf-c-description-list__term--FontSize: var(--pf-global--FontSize--sm);
   --pf-c-description-list__term--LineHeight: var(--pf-global--LineHeight--sm);
 
+  // icon
+  --pf-c-description-list__term-icon--MinWidth: var(--pf-global--icon--FontSize--sm);
+  --pf-c-description-list__term-icon--MarginRight: var(--pf-global--spacer--sm);
+  --pf-c-description-list__term-icon--FontSize: var(--pf-global--icon--FontSize--sm);
+  --pf-c-description-list__term-icon--Color: var(--pf-global--icon--Color--light);
+
+  // term text
+  --pf-c-description-list__term-text--Display: inline;
+  --pf-c-description-list__term-text--sm--Display: flex;
```

Comment:
If someone has passed multiple elements as the term, switching this to a `flex` layout is going to put those elements in a flex row. I think we need to preserve the `__term-text` element's layout to avoid a breaking change since that's what the children of `<DescriptionListTerm />` are currently passed to. 

https://github.com/patternfly/patternfly-react/blob/b30c7c3f64a443451b47ca5ab30a8a13de521bdd/packages/react-core/src/components/DescriptionList/DescriptionListTerm.tsx#L17-L19

2 thoughts on that. Can we move the icon element to be adjacent to the text element? Otherwise, we could add an opt-in class like `.pf-m-icon` to the text element if there is an icon, then create a new element for the text content, though I think the previous approach would be simpler.

Reply:
@mceledonia @mcarrano Can you verify what you want for the icon size given the discussion in the issue?

Reply:
Agreed with Coker, let's go with 14px font size!